sabnzbd/filesystem.py

```diff
@@ -300,16 +300,16 @@ def create_real_path(name, loc, path, umask=False, writable=True):
     if path:
         my_dir = real_path(loc, path)
         if not os.path.exists(my_dir):
-            logging.info("%s directory: %s does not exist, try to create it", name, my_dir)
+            logging.info("{} directory: {} does not exist, try to create it".format(name, my_dir))
             if not create_all_dirs(my_dir, umask):
-                logging.error(T("Cannot create directory %s"), clip_path(my_dir))
```

Comment:
As @thezoggy mentioned, the `T` function is patched in `lang.py` so it's available everywhere as a more general way to translate instead of Python's weird `_` function. So we really need it!

Reply:
> A lot of these changes break Sab. The T variable is used for translation (language) support.

Thanks @thezoggy and @Safihre, I will return the T variable back to its original position, lol.  I swear I did search and couldn't find a reference anywhere for it, but that makes way more sense.  

Ignore this update for now and I'll take your feedback into account and add a new commit that deals with this.